backend/workflow_manager/endpoint_v2/destination.py

```diff
@@ -176,10 +177,8 @@ def handle_output(
         if connection_type == WorkflowEndpoint.ConnectionType.FILESYSTEM:
             self.copy_output_to_output_directory()
         elif connection_type == WorkflowEndpoint.ConnectionType.DATABASE:
-            if (
-                file_hash.file_destination
-                == WorkflowEndpoint.ConnectionType.MANUALREVIEW
-            ):
+            result = self.get_result(file_history)
+            if WorkflowUtil.db_rule_check(result, workflow, file_hash.file_destination):
```

Comment:
@vishnuszipstack rather than returning `False` by default on such functions which are obscure / not related to OSS - I suggest in either renaming it so that it makes sense for OSS as well or using a proper plugin here. We can't tell what's happening without stepping into the code and seeing such a function `WorkflowUtil.db_rule_check` that always returns False will be confusing to an OSS user

Reply:
@chandrasekharan-zipstack renamed function.